pyproject.toml

```diff
@@ -72,6 +72,7 @@ pytest-cov = ">=4.0"
 pytest-mock = ">=3.9"
 pytest-randomly = ">=3.12"
 pytest-xdist = { version = ">=3.1", extras = ["psutil"] }
+urllib3 = "<2.3"  # see https://github.com/gabrielfalcao/HTTPretty/issues/484
```

Comment:
this crisis was bound to come: I would not bet on there being a new httpretty any time soon...

Reply:
Fortunately, it is only a dev dependency. That is why I would say it is not that relevant for the release. In the medium term, of course, we have to replace it.

Reply:
Indeed httpretty seems stalled. Some projects seems to move their dependency from httpretty to mocket which seems to have mocks specifically aiming at httpretty compatibility.